privacy-snapshot.json

```diff
@@ -46,6 +48,7 @@
   "metamask-sdk.api.cx.metamask.io",
   "metamask.eth",
   "metamask.github.io",
+  "metamask.io",
```

Comment:
Just a question: is this needed since we mock the response in our tests? Or do mocked responses still trigger this?

Reply:
mocked responses do still appear in the privacy snapshot file